src/api/user/test/user.test.js

```diff
@@ -0,0 +1,375 @@
+    lastName: 'Galilei',
+    displayName: 'Galileo Galilei',
+    isAdmin: true,
+    isFlagged: true,
+    feeds: ['https://dev.to/feed/galileogalilei'],
+    github: {
+      username: 'galileogalilei',
+      avatarUrl:
+        'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    },
+  };
+  const user = new User(body);
+  user.created = new Date().toLocaleString();
+
+  const response = await request(app).post('/user').set('Accept', 'application/json').send(user);
+
+  return response;
+};
+
+const postUser2 = async () => {
+  const body = {
+    id: 10002,
+    firstName: 'Carl',
+    lastName: 'Sagan',
+    displayName: 'Carl Sagan',
+    isAdmin: true,
+    isFlagged: true,
+    feeds: ['https://dev.to/feed/carlsagan'],
+    github: {
+      username: 'carlsagan',
+      avatarUrl:
+        'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    },
+  };
+  const user = new User(body);
+  user.created = new Date().toLocaleString();
+  const response = await request(app).post('/user').set('Accept', 'application/json').send(user);
+
+  return response;
+};
+
+// Tests
+describe('GET REQUESTS', () => {
+  beforeEach(async () => {
+    await clearData();
+  });
+
+  afterEach(async (done) => {
+    await clearData();
+    done();
+  });
+
+  test('Get JSON, get all users', async () => {
+    await postUser1();
+    await postUser2();
+    const response = await getUsers();
+    expect(response.statusCode).toBe(200);
+    expect(response.body.length).toBe(2);
+    // expect(response.body).toStrictEqual([
+    //   {
+    //     id: 10001,
+    //     firstName: 'Galileo',
+    //     lastName: 'Galilei',
+    //     displayName: 'Galileo Galilei',
+    //     isAdmin: true,
+    //     isFlagged: true,
+    //     feeds: ['https://dev.to/feed/galileogalilei'],
+    //     github: {
+    //       username: 'galileogalilei',
+    //       avatarUrl:
+    //         'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    //     },
+    //     created: response.body.created, // ...
+    //   },
+    //   {
+    //     id: 10002,
+    //     firstName: 'Carl',
+    //     lastName: 'Sagan',
+    //     displayName: 'Carl Sagan',
+    //     isAdmin: true,
+    //     isFlagged: true,
+    //     feeds: ['https://dev.to/feed/carlsagan'],
+    //     github: {
+    //       username: 'carlsagan',
+    //       avatarUrl:
+    //         'https://avatars.githubusercontent.com/u/7242003?s=460&u=733c50a2f50ba297ed30f6b5921a511c2f43bfee&v=4',
+    //     },
+    //     created: response.body.created, // ...
+    //   },
+    // ]);
+  });
+
+  test('Get JSON, Accepted - postUser() was called, thus user 10001 exists', async () => {
+    await postUser1();
+    const response = await getUser();
+    expect(response.statusCode).toBe(200);
+    // expect(response.body).toStrictEqual({
```

Comment:
And here?

Reply:
I think this grabs a user, and if it does so successfully, it'll return a code of `200`, which is what @chrispinkney  is checking for